src/Home.js

```diff
@@ -2,16 +2,27 @@ import React from 'react';
 import ModulePlan from './ModulePlan';
 import PlanningSection from './PlanningSection';
 import moduleplan from './moduleplan.json';
+import users from './users.json';
 
 class Home extends React.Component {
   render() {
```

Comment:
Implement the changes, but the new function is included in the Home.js for now. We probably want to move it later in another file?